frontend/src/libs/helpers/conditionally-delete-or-update/conditionally-delete-or-update.helper.ts

```diff
@@ -0,0 +1,20 @@
+type ItemType = {
+  id: number;
+};
+
+const conditionallyDeleteOrUpdate = <T extends ItemType>(
+  items: T[],
+  itemToDeleteOrUpdate: T,
+  hasToDelete: boolean,
+): T[] => {
```

Comment:
Was not resolved - pass arguments as object
```suggestion
const conditionallyDeleteOrUpdate = <T extends ItemType>({
  items,
  itemToDeleteOrUpdate,
  hasToDelete,
}: Args<T>): T[] => {
```